src/actions/actionCreators/favoritesBeerList.js

```diff
@@ -0,0 +1,35 @@
+import { actionTypes } from '../actionTypes';
+import * as api from '../../utils/api';
+import { isFetching } from '../../reducers/favoritesBeerList';
+
+export const fetchBeers = (onSuccess) => (dispatch, getState) => {
+    const state = getState();
+
+    if(isFetching(state)) {
+        return;
+    }
+
+    dispatch(requestBeers());
+
+    const { favoritesRequest, favorites } = state;
+
+    return api.fetchBeers(favoritesRequest).then(response => 
+        dispatch(receiveBeers(onSuccess(response, favorites.beerIds)))
```

Comment:
There is not big difference between fetch and receive here. Let's do stress on type of result and name this action creator `succeedFetchingBeers`.
`failFetchingBeer`.